FIAT/functional.py

```diff
@@ -71,39 +71,18 @@ def __init__(self, ref_el, target_shape,
         return
 
     def evaluate(self, f):
-        """Evaluates the functional on some callable object f."""
-        result = 0
-
-        # non-derivative part
```

Comment:
In particular, do we really need different evaluation methods for different functional types? It seemed to me that `pt_dict` and `deriv_dict` are sufficient for the evaluation of a functional, aren't they?